AGENTS.md

```
@@ -28,4 +28,11 @@
 ### Architecture
 - One resource per RPC call (no batching)
 - Region-specific binaries with build tags
-- Thread-safe pricing lookups for concurrent gRPC calls
\ No newline at end of file
+- Thread-safe pricing lookups for concurrent gRPC calls
+
+## Active Technologies
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)